version/version.go

```diff
@@ -0,0 +1,29 @@
+package version
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// This module provides the ability to inject Build (git sha) and Version information at compile time.
+// To set these values invoke go build as follows
+
+// go build -ldflags “-X github.com/flyteorg/flyteadmin/version.Build=xyz -X github.com/flyteorg/flyteadmin/version.Version=1.2.3"
```

Comment:
very cool! can we add this here https://github.com/flyteorg/flyteadmin/blob/master/Makefile#L28 so it gets used in the release image https://github.com/flyteorg/flyteadmin/blob/master/Dockerfile#L21

Reply:
@evalsocket @katrogan, why do we need to copy this here? This exists in flytestdlib, can we not just reuse that?

Reply:
oh I did not know this! Yes let's re-use it